CppCoreGuidelines.md

```diff
@@ -1223,12 +1223,12 @@ Correctness. Assumptions not stated in an interface are easily overlooked and ha
 
 Controlling the behavior of a function through a global (namespace scope) variable (a call mode) is implicit and potentially confusing. For example:
 
-    int rnd(double d)
+    int round(double d)
     {
-        return (rnd_up) ? ceil(d) : d;    // don't: "invisible" dependency
+        return (round_up) ? ceil(d) : d;    // don't: "invisible" dependency
```

Comment:
it would be even more clearer if the variable round_up was declared. For example above the function. A comment to the variable explaining it's purpose, would let a novice understand easier.